apps/docs/content/pages/getting-started.mdx

```diff
@@ -0,0 +1,187 @@
+---
+title: Getting started
+description: Getting started with Workleap Design Tokens
+---
+
+This guide will help you get started with Hopper UI. Hopper UI is a Design System that provides a collection of components, tokens, and icons to help you build applications.
```

Comment:
I'm not sure how to word this properly, but i feel like this page is really really too techy, and not consumer oriented. 

There is no such thing as a "Quick Install". 

There are 2 main use case: 
- You're a react dev
- You're a non react dev.

It should be reflected straight away, and maybe redirected to a new page, such as getting-started/react. 

So `/getting-started`

This guide will help you get started with Hopper UI. Hopper UI is a Design System that provides a collection of components, tokens, and icons to help you build applications.

# Installation Guide

Follow one of the guides below to get started with Hopper. 

[ React Application ] [ JavaScript Application ] [ Individual Packages Installation]

# Contribute

Pull requests are welcome. For major changes, please open a [discussion](https://github.com/gsoft-inc/wl-hopper/discussions/new/choose) first to discuss what you would like to change. If you're interested in contributing, check out our [Contributing Guide](https://github.com/gsoft-inc/wl-hopper/blob/main/CONTRIBUTING.md)!
  1 change: 1 addition & 0 deletions1  
[apps/docs/package.json](https://github.com/gsoft-inc/wl-hopper/pull/254/files#diff-d03d2ce97fad5059ad6248fca5f5919994b09bca360f883318c03f1f817ae677)

Then `/getting-started/react`

<Callout>Hopper is compatible with React 18. Not using React, or using an older version than 18? You can still benefit from Hopper, read on to discover how. LINK TO INDIVIDUAL PACKAGE INSTALLATION REACT 16 ICONS</Callout>

La section qui est dans quick install ici


Then `/getting-started/javascript`

la section tokens + svg-icons mergé

Then `/getting-started/individual-packages`

tt les installations

Reply:
I agree to an extent, I feel that adding "hub" page is an unnecessary click in most cases. How about /getting-started would be the default installation aka react?

Reply:
sure ya